cli/ioctl/cmd/action/actiondeploy.go

```diff
@@ -7,55 +7,37 @@
 package action
 
 import (
-	"fmt"
 	"math/big"
 
-	"github.com/spf13/cobra"
-	"go.uber.org/zap"
-
-	"github.com/iotexproject/iotex-core/action"
 	"github.com/iotexproject/iotex-core/cli/ioctl/util"
-	"github.com/iotexproject/iotex-core/pkg/log"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
 )
 
 // actionDeployCmd represents the action deploy command
 var actionDeployCmd = &cobra.Command{
-	Use:   "deploy -s SIGNER -b BYTE_CODE -l GAS_LIMIT [-p GAS_PRICE]",
+	Use:   "deploy [AMOUNT_IOTX] -s SIGNER -b BYTE_CODE [-l GAS_LIMIT] [-p GAS_PRICE]",
```

Comment:
`[AMOUNT_IOTX]` becomes a must value to fill?

Reply:
default value is 0, we may leave it as optional.